dbfit-java/derby/src/main/java/dbfit/environment/DerbyEnvironment.java

```diff
@@ -84,7 +85,80 @@ public Pattern getParameterPattern() {
 
     public Map<String, DbParameterAccessor> getAllProcedureParameters(
             String procName) throws SQLException {
-        throw new UnsupportedOperationException();
+        String[] qualifiers = procName.split("\\.");
```

Comment:
Really? I could trim it down to 80 chars (or so). It's a bit 1970s green-on-black style though! What do you think the limit should be?

Personally I find split method parms and arguments split over lines to not look good.